src/Domain/Codec/BytesBinaryCodec.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Erdcsharp.Domain.Helper;
+using Erdcsharp.Domain.Values;
+
+namespace Erdcsharp.Domain.Codec
+{
+    public class BytesBinaryCodec : IBinaryCodec
+    {
+        public string Type => TypeValue.BinaryTypes.Bytes;
+
+        private const int BytesSizeOfU32 = 4;
+
+        public (IBinaryType Value, int BytesLength) DecodeNested(byte[] data, TypeValue type)
+        {
+            var sizeInBytes = (int)BitConverter.ToUInt32(data.Take(BytesSizeOfU32).ToArray(), 0);
+            if (BitConverter.IsLittleEndian)
```

Comment:
Good, now we need two types of machines to test this code :nerd_face: 

I see there are some utilities in .NET for reading, say, a U32 big-endian, but it doesn't seem to be in .NET standard 2.0, right?

https://docs.microsoft.com/en-us/dotnet/api/system.buffers.binary.binaryprimitives.readint32bigendian